src/platforms/common/performance/index.mdx

```diff
@@ -84,6 +84,19 @@ Leaving the sample rate at `1.0` means that included instrumentation will send a
 
 </PlatformSection>
 
+<!-- notSupported here should be the opposite of whats in common/connect-services.mdx -->
+<PlatformSection notSupported={["javascript", "javascript.cordova"]}>
+
+## Connecting Services
+
+If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) depending on where your request originates there are two methods to connect your traces:
+1. For requests that start in your backend, this is done by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
+2. For requests that start in JavaScript, this is done by [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.
+
+Otherwise backend services with Performance Monitoring will connect automatically.
```

Comment:
The setting of the header for outgoing requests is automatic, and influenced by `tracingOrigins`.

Isn't (2) sort of suggesting some manual work? (1) is definitely manual/intentional work.

Reply:
Hmm good point, I updated this to callout specifically that the SDK will add the header for you, and kept the link so they can click to learn more if for whatever reason it doesn't